tests/keras/layers/convolutional_test.py

```diff
@@ -72,8 +72,9 @@ def test_conv_1d():
     input_dim = 2
     kernel_size = 3
     filters = 3
+    paddings = _convolution_paddings + ['causal'] if K.backend() == 'mxnet' else _convolution_paddings
```

Comment:
shall we change to logic to: add 'causal' to _convolution_paddings if K.backend() != 'theano' ? Now only theano does not support causal padding.

Reply:
Addressed comment
Didn't add this earlier as this was also modifying the way tensorflow unit tests causal conv1d